pkg/repository/v1_repository.go

```diff
@@ -94,12 +94,14 @@ func (r *V1Repository) UpdateComponents(specs []ComponentSpec) error {
 		}
 
 		if spec.Nightly {
-			spec.Version = manifest.Nightly
+			spec.Version = "nightly"
```

Comment:
Why change the constant definition to string literal?

Reply:
The spec.Version will be the profile directory name to store the component. If use manifest.Nightly, the directory will be something like `vx.x.x-nightly+mmyydd` instead of just "nightly"

Reply:
The real version is stored in specVersion